environs/context/callcontext.go

```diff
@@ -14,7 +16,8 @@ type ModelCredentialInvalidator interface {
 // CallContext creates a CloudCallContext for use when calling environ methods
 // that may require invalidate a cloud credential.
 func CallContext(ctx ModelCredentialInvalidator) *CloudCallContext {
-	callCtx := NewCloudCallContext()
+	// TODO(wallyworld) - pass in the stdcontext
```

Comment:
Not yet. This is called from various facade methods, just once in the method. It would be a lot more churn to pass in a context and unlike the workers where we were setting up a static context and reusing it, it's ok to leave this for now and follow up later to tidy up.